docs/user/integration/dynatrace/README.md

```diff
@@ -0,0 +1,160 @@
+- Deploy [Dynatrace Operator](https://github.com/Dynatrace/dynatrace-operator) (Recommended)
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace you want to use for Dynatrace as a variable. Replace the `{NAMESPACE}` placeholder in the following command and run it:
+
+    ```bash
+    export DYNATRACE_NS="{NAMESPACE}"
+    ```
+1. If you haven't created a Namespace yet, do it now:
+    ```bash
+    kubectl create namespace $DYNATRACE_NS
+    ```
+
+### Create access token
+
+Before connecting Kyma and Dynatrace, create a Dynatrace access token:
+
+1. In the Dynatrace navigation, go to the **Manage** > **Access tokens** > **Generate new token**.
+1. Type the name you want to give to this token. If needed, set an expiration date.
+1. Select the following scopes:
+   - **Ingest metrics**
+   - **Ingest OpenTelemetry traces**
+1. Click **Generate token**.
+1. Copy and save the generated token.
+
+### Create Secret
+
+1. To create a new secret containing your access token. In the following command, replace the `{API_TOKEN}` placeholder with the previously created token and run it::
+    ```bash
+    kubectl -n $DYNATRACE_NS create secret generic dynatrace-token --from-literal="apiToken=Api-Token {API_TOKEN}"
+    ```
+
+### Ingest Traces
+
+#### Enable Collecting Istio Traces
+
+By default, the tracing feature of the Istio service mesh is disabled to save resources for the case that no TracePipeline is defined.  To enable emitting traces with the kyma-traces Istio extension provider, create a Telemetry custom resource in the `istio-system` Namespace:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+    randomSamplingPercentage: 1.0
+EOF
+```
+
+This configuration samples 1% of all requests. if you want to change the sampling rate, adjust the **randomSamplingPercentage** property.
+
+#### Create Kyma TracePipeline
+
+1. Replace the `{ENVIRONMENT_ID}` placeholder with the environment Id of your Dynatrace instance:
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    kind: TracePipeline
+    metadata:
+        name: dynatrace
+    spec:
+        output:
+            otlp:
+                endpoint:
+                    value: https://apm.cf.eu12.hana.ondemand.com/e/{ENVIRONMENT_ID}/api/v2/otlp
+                headers:
+                    - name: Authorization
+                      valueFrom:
+                          secretKeyRef:
+                              name: dynatrace-token
+                              namespace: ${DYNATRACE_NS}
+                              key: apiToken
+                protocol: http
+    EOF
+    ```
+1. To find traces from your Kyma cluster in the Dynatrace UI, go to **Applications & Microservices** > **Distributed traces**.
+
+### Ingest Metrics
```

Comment:
Maybe we can add here a comment that configurable metrics are still in development. That would allow us to publish the document already for tracing.